### PR TITLE
Adding measurement

### PR DESCRIPTION
**Pull request by get-itips**
https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/issues/4727

@LolaJacobsen please review

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/dimensions-and-measures-available-in-call-quality-dashboard.md [hunk 1]
```diff
@@ -510,6 +510,7 @@ Many Measurement values can also be used as filters. The following table lists t
 |Total Answer Seizure Ratio |Ratio |Ratio of calls with duration less than 5 seconds over the total number of calls. |
 |Total Short Call Percentage |Percentage |Percentage of total calls less than 1 minute long. |
 |Total Media Failure Percentage |Percentage |Percentage of all streams where either media path could not be established or did not terminate normally. |
+|Total Audio Stream Duration |Minutes |Total Audio Stream Duration in minutes in the selected time range. |
```
Comment (JohanFreelancer9) [inline Teams/dimensions-and-measures-available-in-call-quality-dashboard.md:513]:
```suggestion
|Total Audio Stream Duration |Minutes |Total audio stream duration in minutes in the selected time range. |
```

**Comment by scanum** [comment]
@LolaJacobsen PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by LolaJacobsen** [comment]
@GageAmes Please review. OK to merge this PR?

File: Teams/dimensions-and-measures-available-in-call-quality-dashboard.md [hunk 2]
```diff
@@ -510,6 +510,7 @@ Many Measurement values can also be used as filters. The following table lists t
 |Total Answer Seizure Ratio |Ratio |Ratio of calls with duration less than 5 seconds over the total number of calls. |
 |Total Short Call Percentage |Percentage |Percentage of total calls less than 1 minute long. |
 |Total Media Failure Percentage |Percentage |Percentage of all streams where either media path could not be established or did not terminate normally. |
+|Total Audio Stream Duration |Minutes |Total audio stream duration in minutes in the selected time range. |
```
Comment (GageAmes) [inline Teams/dimensions-and-measures-available-in-call-quality-dashboard.md:513]:
```suggestion
|Total Audio Stream Duration (Minutes) |Minutes |Total audio stream duration in minutes in the selected time range. |
```
The actual measurement name is "Total Audio Stream Duration (Minutes)"

**Pull request merged**